### PR TITLE
add-labels-for-etcd-monitoring

### PR DESCRIPTION
**Pull request by calvix**
add labels that are needed for Prometheus config controller in order to support etcd monitoring

towards
https://github.com/giantswarm/giantswarm/issues/3248#issuecomment-403785899

### CODE REVIEW

File: service/controller/v14/resource/service/master_service.go [hunk 1]
```diff
@@ -20,6 +20,10 @@ func newMasterService(customObject v1alpha1.KVMConfig) *apiv1.Service {
 				"cluster":  key.ClusterID(customObject),
 				"customer": key.ClusterCustomer(customObject),
 				"app":      key.MasterID,
+				"giantswarm.io/cluster":        key.ClusterID(customObject),
+				"giantswarm.io/etcd_domain":    key.ClusterEtcdDomain(customObject),
+				"giantswarm.io/organization":   key.ClusterCustomer(customObject),
+				"giantswarm.io/version_bundle": key.VersionBundleVersion(customObject),
```
Comment (calvix) [inline service/controller/v14/resource/service/master_service.go]: aligning with aws operator

Comment (xh3b4sd) [reply service/controller/v14/resource/service/master_service.go]: Can we please have constants in the key package and use these here? IMO the workers should have these labels as well.

Comment (calvix) [reply service/controller/v14/resource/service/master_service.go]: Why do you think that worker should have the labels?

Comment (xh3b4sd) [reply service/controller/v14/resource/service/master_service.go]: I thought our labels should be on all resources we manage.

**Pull request merged**